strelka.wdl

```diff
@@ -4,25 +4,27 @@ import "common.wdl" as common
 
 task Germline {
     input {
-        String runDir
+        String runDir = "."
```

Comment:
This seems like a bad idea, creating a dedicated directory would likely be much safer.
```suggestion
        String runDir = "./strelka_run"
```